numpy/core/src/umath/svml/svml_z0_acos_d_la.s

```diff
@@ -0,0 +1,2642 @@
+/*******************************************
+* Copyright (C) 2021 Intel Corporation
+* SPDX-License-Identifier: BSD-3-Clause
+*******************************************/
+
+
+	.text
+.L_2__routine_start___svml_acos8_z0_0:
+
+	.align    16,0x90
+	.globl __svml_acos8
+
+__svml_acos8:
+
+
+	.cfi_startproc
+..L2:
+
+        pushq     %rbp
+	.cfi_def_cfa_offset 16
+        movq      %rsp, %rbp
+	.cfi_def_cfa 6, 16
+	.cfi_offset 6, -16
+        andq      $-64, %rsp
+        subq      $192, %rsp
+        vmovups   __svml_dacos_data_internal(%rip), %zmm7
+        vmovups   64+__svml_dacos_data_internal(%rip), %zmm8
+
+/* S ~ 2*sqrt(Y) */
+        vmovups   128+__svml_dacos_data_internal(%rip), %zmm11
```

Comment:
The SVML files are cloned from the ICC complier and each umath function is well contained within a single file. Since they aren't really used anywhere else (at least as of now) I would prefer to leave them as they are. If and when we convert this to universal intrinsic format, we could rewrite the algo using intrisics and eventually just get rid of the asm file all together.

Reply:
To get it clear, the original code has been written by VLANG or C/C++, we're dealing with disassembled version licensed by Intel and there's no intention from Intel to share the original source. By quick estimating the cost of efforts to make this code maintainable, more readable, and support wide assemblers. it will reward the time spent converting it into SIMD/universal intrinsics. So yes, "leave them as they are" is the only possible choice.